tests/integration/security/mtlsk8sca/strict_test.go

```diff
@@ -33,6 +34,7 @@ import (
 // - Send HTTP/gRPC requests between apps.
 func TestMtlsStrictK8sCA(t *testing.T) {
 	framework.NewTest(t).
+		Features(features.Security_Mtls_K8sCa).
```

Comment:
```suggestion
		Features("security.mtls.k8s-ca").
```

Reply:
This looks great!  I've adjusted your feature label to reflect that there are no constants generated in the features package.  Any remaining constants should be removed, and I'll try to get to that tomorrow.